lib/screens/explore_screens/regions_screens/regions_main/region_screen.dart

```diff
@@ -73,19 +70,17 @@ class RegionScreen extends StatelessWidget {
 }
 
 class _SliverAppBarDelegate extends SliverPersistentHeaderDelegate {
-  _SliverAppBarDelegate(this._tabBar);
-
-  final TabBar _tabBar;
+  const _SliverAppBarDelegate();
 
   @override
-  double get minExtent => _tabBar.preferredSize.height;
+  double get minExtent => 48;
```

Comment:
can you leave a comment explaining why we set this card coded to a number? 
Can you move this number to a constant with a meaningful name. 
NO magic numbers pls

Reply:
moved as a constant